proposals/0000-things-to-improve.md

```diff
@@ -0,0 +1,69 @@
+<!-- {% raw %} -->
+
+# List of Things to Improve in the HLSL Language
+
+* Proposal: [NNNN](NNNN-things-to-improve.md)
+* Author(s): [Helena Kotas](https://github.com/hekota)
+* Status: **Design In Progress**
+
+## Introduction
+
+This document includes a list of features that we might consider removing in a future
+version of the HLSL language. It includes things that:
+- have unusual syntax or odd behavior
+- are remnants of outdated features that are no longer used or relevant
+- are construct that are generally considered "not very nice" from a language point of view
+
+## Motivation
+
+We strive to evolve HLSL into a modern shader language. In order to so we need to recognize
+odd, outdated or obscure features that are no longer used or that we do not wish to support
+going forward.
+
+## List of Things to Improve
+
+### 1. Global variables outside of `cbuffer` context
+
+All global variables declared outside of `cbuffer` context go implicitly into `$Globals` constant buffer. If the variable has an initializer it gets ignored.
```

Comment:
> If the variable has an initializer it gets ignored.

A little ambiguous whether "it" refers to the variable or the initializer.  Maybe "Any initializers on global variables are ignored".